elm-package.json

```diff
@@ -8,15 +8,22 @@
     ],
     "exposed-modules": [
         "Core.NotificationDrawer",
-        "Core.NotificationMenu"
+        "Core.NotificationMenu",
+        "Core.Data.Make",
+        "Core.Data.Model",
+        "Core.MakeModelMenu",
+        "Core.Modal",
+        "Core.ModalPorts"
```

Comment:
You can't do this, please run `elm make` and see if it compiles, I'd be surprised if it let you